cog.yaml

```diff
@@ -0,0 +1,38 @@
+# Configuration for Cog ⚙️
+# Reference: https://github.com/replicate/cog/blob/main/docs/yaml.md
+
+build:
+  # set to true if your model requires a GPU
+  gpu: true
+
+  # a list of ubuntu apt packages to install
+  system_packages:
+    - "ffmpeg"
+    - "libsndfile1"
+
+  # python version in the form '3.8' or '3.8.12'
+  python_version: "3.9"
+
+  # a list of packages in the format <package-name>==<version>
```

Comment:
@daanelson is there a way to point to a file like requirements.txt or a locked version of it?

Reply:
@hmartiro unfortunately there's not a clean approach. Cog uses the info in the `python_packages` section to figure out which nvidia base image & version of CUDA to use when building the container and there's not a way to pass in a `requirements.txt` instead. The goal is to keep all the info needed in `cog.yaml` for the build, though point taken that passing a `requirements.txt` would be useful in some cases (especially for folks using poetry, etc.). 

You can run arbitrary commands (including `curl`-ing and then installing some arbitrary `requirements.txt`) in the `run` section, though b/c of how cog solves for CUDA versions you can't do this for all your dependencies, so imo adds more complexity than it's worth.